src/pudl/transform/eia860.py

```diff
@@ -108,12 +101,21 @@ def ownership(eia860_dfs, eia860_transformed_dfs):
     own_df = pd.concat([without_known_dupes, deduped])
     # Check whether we have truly deduplicated the dataframe.
     remaining_dupes = own_df[own_df.duplicated(subset=own_pk, keep=False)]
-    if len(remaining_dupes) > 0:
+    if not remaining_dupes.empty:
         raise ValueError(
             "Duplicate ownership slices found in ownership_eia860:"
             f"{remaining_dupes}"
         )
 
+    # Remove a couple of records known to have null values in the primary key
+    mask = (
+        (own_df.report_date.isin(["2018-01-01", "2019-01-01"]))
```

Comment:
You could create variables for the targeted drop values so it's easy to add more in the future.

Reply:
The problem we've had with doing this in the past is that the method of selection often doesn't apply to a large class of records. Like there could be other records with nulls in the primary key columns, but maybe the null is in `owner_utility_id_eia` rather than `generator_id`.

Setting the index to be the primary key columns and then storing the index values of records that need to be dropped along with a comment as to why would work well. I tried this here, but it failed because if you try and drop by index value and that index value isn't present, it fails -- and that happens here because in the tests we're only processing 2019 data, but there are things that need to be dropped from other years too when they're present. So I reverted to the masking method.

Another complication in the case of duplicate values in the primary keys is that we may need to use different methods to choose which of the duplicates we want to retain, if any.  Or if we instead want to aggregate the duplicates somehow.

So I think I'm inclined not to create a structure to hold this information until we see a generalizable pattern.

Reply:
Sounds good 👍